jenkins_python/scripts/PullRequestReport.py

```diff
@@ -201,6 +267,9 @@ def buildTestReport(templateEnv):
                          description='Finished at ' + time.strftime("%d %b %Y %H:%M GMT"), context='Pylint')
 lastCommit.create_status(state=statusMap[failedUnitTests]['ghStatus'], target_url=reportURL + '#unittests',
                          description='Finished at ' + time.strftime("%d %b %Y %H:%M GMT"), context='Unit tests')
+lastCommit.create_status(state=statusMap[failedPy27]['ghStatus'], target_url=reportURL + '#py27',
```

Comment:
Let's call this pyfuture for the A tag and "Python3 compatibility" for the GitHub context.